src/components/UI/ReadMore/ReadMore.js

```diff
@@ -0,0 +1,19 @@
+import PropTypes from 'prop-types';
+
+import styles from './ReadMore.module.scss';
+
+const READ_MORE_TXT = 'read more';
```

Comment:
why not in strings?

Reply:
Because its a preparation for taking out generic components to a separate lib it's not in strings.js, 
I put it inside of ReadMore.strings.js, and @dan-ziv responded to me with this:
`We're not using this convention anymore.
Just create const var inside this file which defines the text.
const READ_MORE_TXT = '....'`